rclcpp/include/rclcpp/subscription_options.hpp

```diff
@@ -51,6 +52,24 @@ struct SubscriptionOptionsBase
   /// Optional RMW implementation specific payload to be used during creation of the subscription.
   std::shared_ptr<rclcpp::detail::RMWImplementationSpecificSubscriptionPayload>
   rmw_implementation_payload = nullptr;
+
+  // Options to configure topic statistics collector in the subscription
```

Comment:
nitpick: use punctuation, following the style in the rest of the file, here and in other parts of your changes